quickstarts/contentsquare/config.yml

```diff
@@ -7,18 +7,26 @@ title: Contentsquare
 
 # Displayed in search results and recommendations. Summarizes a quickstarts functionality.
 summary: |
+  ## Contentsquare performance monitoring
```

Comment:
Seems like this is showing up funky on the preview. Perhaps you should just remove it as I don't think it's necessary.

Reply:
Good catch @brammerl!
@aliandersonsn, the summary field does not support markdown. I'm thinking we could probably just remove this title line since it the display name of the quickstart will be right above it!